arm-storage/2017-06-01/swagger/storage.json

```diff
@@ -20,6 +20,28 @@
     "text/json"
   ],
   "paths": {
+    "/providers/Microsoft.Storage/operations":{
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "description": "Lists all of the available Storage Rest API operations.",
+        "operationId": "Operations_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/OperationListResult"
```

Comment:
Basd on the schema of `OperationListResult` model, seems like this operation should have [x-ms-pageable ](https://github.com/Azure/autorest/tree/master/docs/extensions#x-ms-pageable) extension so that SDK can undersand and leverage the best paging mechanism on client side.